src/codeu/chat/common/BasicController.java

```diff
@@ -51,4 +51,13 @@
   //  Whether conversations can have the same title is undefined.
   ConversationHeader newConversation(String title, Uuid owner);
 
+  // NEW INTEREST
+  //
+  //   Create a new interest on the server. All parameters must be provided
+  //   or else the server won't apply the change. If the operation is
+  //   successful, an Interests object will be returned representing the full
+  //   state of the interests of the specified user on the server.
+  // TODO: uncomment when method is added to user Controller
+  //Interests newInterest(Uuid userid, Uuid interest);
```

Comment:
Yes, I did because the user's controller also implements this interface and it doesn't override this function yet.